lms/envs/common.py

```diff
@@ -2147,6 +2147,9 @@
 ENABLE_CREDIT_ELIGIBILITY = True
 FEATURES['ENABLE_CREDIT_ELIGIBILITY'] = ENABLE_CREDIT_ELIGIBILITY
 
+################ Enable JWT auth ####################
+FEATURES['ENABLE_JWT_AUTH'] = False
```

Comment:
please add a comment here to the effect of:  'when this is set to False, API endpoints using JSONWebTokenAuthentication will reject requests using JWT to authenticate, even if those tokens are valid.  Set this to True only if you need those endpoints, and have configured settings.JWT_AUTH to override its default values with secure values.'